docs/contributors/testing-overview.md

```diff
@@ -363,14 +368,19 @@ Part of the unit-tests suite is a set of Jest tests run exercise native-mobile c
 ### Debugging the native mobile unit tests
 
 To locally run the tests in debug mode, follow these steps:
+
 0. Make sure you have ran `npm ci` to install all the packages
 1. Run `npm run test-unit:native:debug` inside the Gutenberg root folder, on the CLI. Node is now waiting for the debugger to connect.
 2. Open `chrome://inspect` in Chrome
-3. Under the "Remote Target" section, look for a `../../node_modules/.bin/jest ` target and click on the "inspect" link. That will open a new window with the Chrome DevTools debugger attached to the process and stopped at the beginning of the `jest.js` file. Alternatively, if the targets are not visible, click on the `Open dedicated DevTools for Node` link in the same page.
+3. Under the "Remote Target" section, look for a `../../node_modules/.bin/jest` target and click on the "inspect" link. That will open a new window with the Chrome DevTools debugger attached to the process and stopped at the beginning of the `jest.js` file. Alternatively, if the targets are not visible, click on the `Open dedicated DevTools for Node` link in the same page.
 4. You can place breakpoints or `debugger;` statements throughout the code, including the tests code, to stop and inspect
 5. Click on the "Play" button to resume execution
 6. Enjoy debugging the native mobile unit tests!
 
+### Native mobile end-to-end tests
+
+Contributors to Gutenberg will note that PR's include continuous integration E2E tests running the native mobile E2E tests on Android and iOS. For troubleshooting failed tests, check our guide on [native mobile tests in continious integration](docs/contributors/native-mobile.md#native-mobile-e2e-tests-in-continuous-integration). More information on running these tests locally can be found in the [relevant directory README.md](https://github.com/WordPress/gutenberg/tree/master/packages/react-native-editor/__device-tests__).
```

Comment:
Not sure, is "PR's" the proper plural (instead of "PRs")? 🤔

Reply:
PRs seemed to be the more common convention, updated! https://github.com/WordPress/gutenberg/pull/23951/commits/1dea34706429bcb9c0643fec1e48fb2f281444eb